spec/models/favorite_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Favorite do
+  describe "#total_count" do
+    it "can calculate the total number of items it holds" do
+      favorite = Favorite.new([1,2,3,4,5])
+
+      expect(favorite.total_count).to eq(5)
```

Comment:
maybe a test for
```
favorite = Favorite.new()

expect(favorite.total_count).to eq(0)
```
to start instead before adding to it. i don't see how a user in a new session would start out with 5 favorites if we haven't implemented cookies yet. although it's nice to see that it does work that way if we need it to.